cockpit-certificates.spec.in

```diff
@@ -16,6 +16,13 @@ BuildRequires:  make
 Requires: cockpit-bridge
 Requires: certmonger
 
+Provides: bundled(nodejs-object-assign) = 4.1.1
```

Comment:
OOI how did you get this list?

Reply:
It's a direct translation of the generated dist/index.js.LICENSE.txt.gz from terser. Arguably this should be autogenerated at some point.